### PR TITLE
DOCS-3513 Using Synthetic Metrics in Monitors Guide

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do?

Adds a guide in the Synthetic Monitoring Guides section.

### Motivation

DOCS-3513

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+title: Using Synthetic Metrics
```
Comment (alai97) [inline content/en/synthetics/guide/using-synthetic-metrics.md:2]:
```suggestion
title: Using Synthetic Metrics in Monitors
```

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+kind: guide
+description: Learn how to use general Synthetic metrics in monitors. 
+further_reading:
+- link: "/monitors/metrics/"
```
Comment (alai97) [inline content/en/synthetics/guide/using-synthetic-metrics.md:6]:
```suggestion
- link: "/monitors/types/metric/"
```

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+{{< img src="synthetics/guide/using-synthetic-metrics/metric-monitor.png" alt="Example metric monitor that alerts when too many tests are failing in CI" style="width:95%;" >}}
+
+With metric monitors, you can monitor the total response time, scope on specific HTTP timings such as DNS, the DNS resolution, and TCP connection, and access tags added to metrics coming from Synthetic tests. 
+
+This guide demonstrates how to set up a metric monitor using a general metric such as `synthetics.test_runs`. 
+
+## Create a metric monitor
+
+You can create monitors to monitor total response time, scope HTTP timings such as DNS resolution, monitor a TCP connection, and more.
```
Comment (estherk15) [inline content/en/synthetics/guide/using-synthetic-metrics.md:29]: Repeats line 23

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+  text: "Learn about Synthetic Monitoring metrics"
+---
+
+## Overview
+
+You can use [metrics][1] generated from your Synthetic tests to create [metric monitors][2] in addition to the [monitor created with your test][3].
+
+{{< img src="synthetics/guide/using-synthetic-metrics/metric-monitor.png" alt="Example metric monitor that alerts when too many tests are failing in CI" style="width:95%;" >}}
+
+With metric monitors, you can monitor the total response time, scope on specific HTTP timings such as DNS, the DNS resolution, and TCP connection, and access tags added to metrics coming from Synthetic tests.
```
Comment (estherk15) [inline content/en/synthetics/guide/using-synthetic-metrics.md:23]: Could a bulleted list work here?

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+  tag: "Documentation"
+  text: "Learn how to manage monitors"
+- link: "/synthetics/metrics/"
+  tag: "Documentation"
+  text: "Learn about Synthetic Monitoring metrics"
+---
+
+## Overview
+
+You can use [metrics][1] generated from your Synthetic tests to create [metric monitors][2] in addition to the [monitor created with your test][3].
```
Comment (estherk15) [inline content/en/synthetics/guide/using-synthetic-metrics.md:19]:
```suggestion
You can use [metrics][1] generated from your Synthetic tests to create [metric monitors][2] in addition to the [Synthetic monitor created with your test][3].
```

**Review by estherk15 — APPROVED** [review]
Just left some suggestions, but looks good to me!

File: content/en/synthetics/guide/using-synthetic-metrics.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+3. In the **Define the metric** section, enter a Synthetic Monitoring metric such as `synthetics.test_runs`, where you can filter on status, response codes, and retry behavior.
+
+4. Set the alerting conditions and add a notification message.
+
+5. Set editing permissions and click **Create**.
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
```
[the remark this answers is not in the document]
Comment (alai97) [reply content/en/synthetics/guide/using-synthetic-metrics.md:61]: It is included in the Further Reading section already!

File: content/en/synthetics/guide/using-synthetic-metrics.md (hunk 4 above)
Comment (alai97) [reply content/en/synthetics/guide/using-synthetic-metrics.md:23]:
```suggestion
With metric monitors, you can accomplish the following:

- Monitor the total response time
- Scope on specific HTTP timings such as DNS, the DNS resolution, and TCP connection
- Access tags added to metrics coming from Synthetic tests 
```

File: content/en/synthetics/guide/using-synthetic-metrics.md (hunk 3 above)
Comment (alai97) [reply content/en/synthetics/guide/using-synthetic-metrics.md:29]:
```suggestion
```

**Pull request merged**